usl_pipeline/usl_lib/usl_lib/readers/config_readers.py

```diff
@@ -21,3 +23,40 @@ def read_rainfall_amounts(rain_fd: TextIO) -> Sequence[float]:
             continue
         entries.append(float(rainfall_line.group(1)))
     return entries
+
+
+def read_key_value_pairs(fd: TextIO) -> dict:
+    """Reads a config file (.txt, .ini, etc) and returns a dictionary of k/v pairs.
+
+    Args:
+      fd: The file containing the key/value pairs
+
+    Returns:
+      A dictionary of the key/values pairs in the file
+    """
+    # Read lines from file first. If the first line is not blank and doesn't look
+    # like a section header, add [DEAULT] section so it is compliant with
+    # what ConfigParser expects
+    lines = fd.readlines()
+    if lines and not lines[0].strip().startswith("["):
```

Comment:
You don't need to do this conditionally -- it'd just be adding an empty DEFAULT section if there is already a section in the ini file.

You can do this in one line with:

```python
config.read_file(itertools.chain(['[DEFAULT]\n'], fd))
```